cetmix_tower_server/models/cx_tower_command_log.py

```diff
@@ -179,24 +179,31 @@ def _command_finished(self):
         """
         # Trigger next flightplan line
         for rec in self:
+            context_timestamp = fields.Datetime.context_timestamp(
+                self, fields.Datetime.now()
+            )
             if rec.plan_log_id:  # type: ignore
                 rec.plan_log_id._plan_command_finished(rec)  # type: ignore
             elif rec.command_status == 0:
                 rec.create_uid.notify_success(
                     message=_(
-                        "Command '%(name)s' finished successfully",
+                        "%(timestamp)s<br/>"
+                        "Command '%(name)s'<br/> finished successfully",
```

Comment:
No need to put <br/> here. Just `Command '%(name)s' finished successfully` as it was before